javascript/ql/test/library-tests/TaintTracking/capture-flow.js

```diff
@@ -17,3 +17,15 @@ function outerMost() {
 }
 
 sink(outerMost()); // NOT OK - but missed
+
+function confuse(x) {
+    let captured;
+    function f() {
+        captured = x;
+    }
+    f();
+    return captured;
+}
+
+sink(confuse('safe')); // OK - but incorrectly flagged
```

Comment:
```suggestion
sink(confuse('safe')); // OK
```

You updated the expected output, but forgot the test comments.